docs/sources/dashboards/variables/inspect-variable/index.md

```diff
@@ -25,6 +25,7 @@ Any variable that is referenced or used has a green check mark next to it, while
 
 ![Variables list](/static/img/docs/variables-templates/variable-not-referenced-7-4.png)
 
-In addition, all referenced variables have a dependency icon next to the green check mark. You can click on the icon to view the dependency map. The dependency map can be moved. You can zoom in out with mouse wheel or track pad equivalent.
+In addition, there is an option to show the dependencies between variables. Before 9.2 all referenced variables have a dependency icon next to the green check mark. You can click on the icon to view the dependency map. After 9.2, it is a button under the list of variables. 
```

Comment:
Are you trying to say that users can choose to have the dependency marker (icon before 9.2 or button after 9.2) shown or not shown? That's what the first sentence seems to suggest. Or are you just trying to indicate that the way you see those dependencies is different from 9.2 on?

```suggestion
In addition, there is an option to show the dependencies between variables. Before 9.2, all referenced variables have a dependency icon next to the green check mark. You can click the icon to view the dependency map. After 9.2, it is a button under the list of variables. 
```